src/handleMessageRequests.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+
+var utils = require("../utils");
+var log = require("npmlog");
+
+module.exports = function(defaultFuncs, api, ctx) {
+  return function handleMessageRequests(threadID, isAccept, callback) {
+    if(!callback && utils.getType(isAccept) === 'Function') {
```

Comment:
I don't really understand why you're making this check in this way.  Why are you checking `!callback`?
Don't you just care about `utils.getType(isAccept) !== 'Boolean'`?
Nitpick: Can we make the name of this function singular?